### PR TITLE
feat(smartcar): Get & Set Charge Limit

### PR DESCRIPTION
**Pull request by EvanPeterson1324**
## Overview
Adds one new data type `VehicleChargeLimit` with two new methods, `getChargeLimit()` and `setChargeLimit(double limit)`

Asana: https://app.asana.com/0/1204073873904879/1204302264096976/f

### Checklist
- [x] Get Charge Limit & javadoc
- [x] Set Charge Limit & javadoc
- [x] Unit tests
- [x] Integration tests

### CODE REVIEW

**Comment by s-ashwinkumar** [comment]
@EvanPeterson1324 The tests are working now. I dont know how this changed but there was no redirect URI in the configs . That said, there is still something you might have to look into in code.
This is the current failure logs : https://app.travis-ci.com/github/smartcar/java-sdk/jobs/599949031
If you scroll down to find the issue you will see this 

```
> Task :compileTestJava FAILED
/home/travis/build/smartcar/java-sdk/src/test/java/com/smartcar/sdk/VehicleTest.java:223: error: unexpected type
    Assert.assertTrue(chargeLimit.getChargeLimit() instanceof Double);
                                                ^
  required: reference
  found:    double
Note: Some input files use or override a deprecated API.
Note: Recompile with -Xlint:deprecation for details.
1 error
```

Points to a code issue that we introduced.

**Comment by EvanPeterson1324** [comment]
> @EvanPeterson1324 The tests are working now. I dont know how this changed but there was no redirect URI in the configs . That said, there is still something you might have to look into in code. This is the current failure logs : https://app.travis-ci.com/github/smartcar/java-sdk/jobs/599949031 If you scroll down to find the issue you will see this
> 
> ```
> > Task :compileTestJava FAILED
> /home/travis/build/smartcar/java-sdk/src/test/java/com/smartcar/sdk/VehicleTest.java:223: error: unexpected type
>     Assert.assertTrue(chargeLimit.getChargeLimit() instanceof Double);
>                                                 ^
>   required: reference
>   found:    double
> Note: Some input files use or override a deprecated API.
> Note: Recompile with -Xlint:deprecation for details.
> 1 error
> ```
> 
> Points to a code issue that we introduced.

I fixed the above issue :+1: I needed to wrap the primitive `double` with the `Double` constructor for it to check properly with `instanceof`

File: src/main/java/com/smartcar/sdk/data/VehicleChargeLimit.java [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  /** @return a stringified representation of VehicleFuel */
+  @Override
+  public String toString() {
+    return this.getClass().getName()
+        + "{"
+        + "limit="
+        + limit
+        + '}';
+  }
+}
```
Comment (EvanPeterson1324) [inline src/main/java/com/smartcar/sdk/data/VehicleChargeLimit.java:25]:
```suggestion
}

```

**Review by s-ashwinkumar — CHANGES_REQUESTED** [review]
Code looks OK to me but you also need to bump the versions. In java sdk it is manually done in the grade properties and readme i think.
Reference PR from last week : https://github.com/smartcar/java-sdk/pull/129/files#diff-3d103fc7c312a3e136f88e81cef592424b8af2464c468116545c4d22d6edcf19L3

**Review by s-ashwinkumar — APPROVED** [review]

**Pull request merged**